README.md

```diff
@@ -3,9 +3,8 @@
 <div align="center">
 
 [![Build Status](https://travis-ci.org/nrwl/nx.svg?branch=master)](https://travis-ci.org/nrwl/nx)
-[![License](https://img.shields.io/npm/l/@nrwl/schematics.svg?style=flat-square)]()
-[![NPM Version](https://badge.fury.io/js/%40nrwl%2Fnx.svg)](https://www.npmjs.com/@nrwl/schematics)
-[![NPM Downloads](https://img.shields.io/npm/dt/@nrwl/schematics.svg?style=flat-square)](https://www.npmjs.com/@nrwl/schematics)
```

Comment:
Is this removed because the number isn't entirely representative of Nx's usage?

Reply:
yup, we point at the deprecated package